ci/taos/plugins-good/pr-prebuild-clang.sh

```diff
@@ -67,7 +67,11 @@ function pr-prebuild-clang(){
 
     # save the result
     clang_format_file="clang-format.patch"
-    git diff > ../report/${clang_format_file}
```

Comment:
For maintenance, please append a prefix  (e.g., clang_option).
* before: option="cxx-only"
* after  : clang_option="cxx-only"

And, we need to append the below statement  in the ./config/config-plugins-prebuild.sh file.
```bash
vi ./config/config-plugins-prebuild.sh 
declare clang_option="" <--- HERE 1/2

. . . Omission . . . 
# clang_option="cxx-only"  <--- HERE 1/2
prebuild_plugins[++idx]="pr-prebuild-clang"
. . . Omission . . . 
```